### ISSUE TITLE
9anime automatic update

### ISSUE THREAD

**Issue by Darkhhour** [matthewdias/kitsu-updater#9]
Hi there, just recently installed kitsu updater and i have a question or two.. on the chrome store it states..

Supported Sites
- 9anime
- AnimaKAI
- Anime Haven*
- Anime Twist
- Crunchyroll*
- Daisuki*
- Funimation*
- GogoAnime
- Hulu*
- KissAnime
- Masterani*
- MoeTube
- Netflix
- SuperAnime
- VIZ*
- VRV*
* These sites do not support automatic updating, and must be manually updated via the popup

Not sure if i am misreading or something has changed but the sites without "*" beside them should auto update without me having to do anything correct? Atleast that's how i read it but it seems to not work on 9anime unless i click the popup and manually input the episodes i watch (which is no different than using kitsu site imo). So am i perhaps doing something wrong and if not is it possible to make 9anime auto update kitsu watch list, it's by far the best site with both dubbed & subbed anime in HD.

Thanks for your hard work.

**Comment by matthewdias** [comment]
You're right, it's supposed to. There are some bugs I need to fix and haven't gotten around to unfortunately.

**Comment by Darkhhour** [comment]
I understand thanks for the update, i want to add that not long after posting this it started auto-updating on 9anime for me but i didn't get the chance to reply until now.
